python/ray/tests/test_placement_group_3.py

```diff
@@ -630,6 +634,8 @@ def is_usage_updated():
         return demand_output["usage"] != ""
 
     wait_for_condition(is_usage_updated)
+
+    time.sleep(AUTOSCALER_UPDATE_INTERVAL_S)
```

Comment:
why don't you just do the whole thing inside wait_for_condition? (verify the cpu usage == expcted)

Reply:
hmm yeah this test could be structured better